src/components/combo/combo.ts

```diff
@@ -325,6 +325,18 @@ export default class IgcComboComponent<T extends object>
     return html`${this.groupKey && item[this.groupKey]}`;
   };
 
+  /**
+   * Sets the component's positioning strategy.
+   */
+  @property({ attribute: 'position-strategy' })
+  public positionStrategy: 'absolute' | 'fixed' = 'fixed';
+
+  /**
+   * Whether the dropdown's width should be the same as the target's one.
+   */
+  @property({ type: Boolean, attribute: 'same-width' })
+  public sameWidth = true;
```

Comment:
This can actually be a bit of an issue - boolean flag true by default I believe is something we avoid, no?

Reply:
Ah, yes. 👍